### ISSUE TITLE
uBlock fails to block elements on browser startup page load. Subsequent page refresh works.

### ISSUE THREAD

**Issue by LittleVulpix** [gorhill/uBlock#3451]
### Describe the issue
When I first open my browser, it navigates to my homepage. This homepage has some elements I want hidden so I have created a filter for them. However, they are not filtered when I first launch my browser. If I press F5 to reload the page, they are blocked, and any subsequent page visit (even in a new tab) works as intended as well. This happens only once during browser startup.

### One or more specific URLs where the issue occurs
https://www.seznam.cz/

### Screenshot in which the issue can be seen
Recorded with licecap on a clean firefox quantum, but the issue manifests on previous firefox and waterfox as well. The section "Další zajímavé články z českého internetu" as well as the little square "Reklama" should both be hidden when the filter is in effect.

https://imgur.com/a/Gfqe8

### Steps for anyone to reproduce the issue

1. Have a clean latest firefox
2. Install uBlock Origin
3. Set www.seznam.cz as your homepage. It will be white; I configured mine as dark to record the video, but the issue manifests just the same on any setting, even if you are logged out. I logged out to test it and to record the video.
4. Open uBlock Origin settings and import my custom filters
5. Restart Firefox - you will see some ad space and the supposedly-filtered content will also be displayed.
6. Refresh the homepage or open it in another tab - filters work as expected now... This leads me to believe it is not a filter issue but a uBlock issue.

### Your settings

- OS/version: Tested on W7 x64 (updated as of January 2018) and W10 x64 Creator's Update
- Browser/version: Tested on Waterfox 56.0.3 and Firefox 56 and Firefox Quantum 57.0.4, same issue on all
- uBlock Origin version: Latest stable offered by updater, 1.14.24.

##### Your filter lists

Default configuration on installation:
uBlock Filters
uBlock Filters - Badware Risks
uBlock Filters - Privacy
uBlock Filters - Resource Abuse
uBlock FIlters - Unbreak
EasyList
EasyPrivacy
Malware Domain List
Malware Domains
Peter Lowe’s Ad and tracking server list

NOTHING ELSE (well, + My filters as seen below)

##### Your custom filters (if any)

There are five custom filters that I use relevant to the page. When testing the bug I removed all other filters to make sure:

! 12/31/2017, 3:00:51 PM https://www.seznam.cz/
www.seznam.cz##.timeline__title-container

! 12/31/2017, 3:01:02 PM https://www.seznam.cz/
www.seznam.cz##.timeline__list

! 12/31/2017, 3:01:07 PM https://www.seznam.cz/
www.seznam.cz##.timeline > ul

! 12/31/2017, 3:01:23 PM https://www.seznam.cz/
www.seznam.cz##.timeline

! 12/31/2017, 3:01:34 PM https://www.seznam.cz/
www.seznam.cz##.ad-button-sticky-repeater__content > .ad-button-sticky-container

**Comment by gorhill** [comment]
Duplicate of #1327.

**Comment by LittleVulpix** [comment]
@gorhill I read through #1327 . Sorry for opening a dupe. I read the filter option setting and you even specifically state to not open an issue about this; sorry.

I set suspendTabsUntilReady to true and everything works as expected now.

Thanks!

**Comment by gorhill** [comment]
Even `suspendTabsUntilReady` is not that reliable, it will work only if uBO is launched before any tab are loaded, which is not guaranteed by Firefox.

**Comment by uBlock-user** [comment]
It is reliable in Chromium though
